tests_config/run_spec.js

```diff
@@ -15,8 +15,11 @@ const prettier = !TEST_STANDALONE
   : require("prettier/standalone");
 
 global.run_spec = (dirname, parsers, options) => {
-  // istanbul ignore next
-  if (!parsers || !parsers.length) {
+  // `isNoParserTests` mean to test `inferParser` on `standalone`
+  const isNoParserTests = dirname.includes("no-parser-tests");
```

Comment:
Let's rename the folder to `parser-inference` to better describe what is actually tested.

Reply:
sure